commands.go

```diff
@@ -1089,13 +1089,13 @@ func (c *commandable) ZRangeWithScores(key string, start, stop int64) *ZSliceCmd
 	return cmd
 }
 
-type ZRangeByScore struct {
+type ZRangeBy struct {
```

Comment:
Please revert this and add a TODO to rename later (in v4). I don't want to break users code compilation.